src/commands/dev/lib/dev-builder.ts

```diff
@@ -82,11 +83,15 @@ async function createBuildProcess(
     nodeBinPromise,
     builderModulePathPromise
   ]);
+  let PATH = `${dirname(execPath)}:${process.env.PATH}`;
```

Comment:
For Windows, it needs to be `${path.delimiter}` instead of `:`

Reply:
@TooTallNate Can this be fixed in a later PR for windows? Surely this isnt the only windows bug and I didn't change this line.

Reply:
Fixed